css-grid-3/Overview.bs

```diff
@@ -873,6 +887,18 @@ Masonry Layout and Placement Algorithm</h3>
 			and then layout the item.
 			Set the [=running position=] of the spanned  [=grid axis=] tracks
 			to <code><var>max_pos</var> + [=outer size=] + 'grid-gap'</code>.
+
+		<li>
+			If the [=masonry container=] uses ''item-pack/dense'' packing,
+			and there exists skipped empty space (e.g. due to spanning items)
```

Comment:
nit: I'd remove this "and" since there is another one later in the list

Reply:
The items in this list are a bit complex, and themselves include commas, so I felt it was easier to understand the sentence with the extra emphasis from the explicit "and" here.

Reply:
I see, yeah that sounds good to me then. Resolving